ct-app/core/node.py

```
@@ -399,79 +404,90 @@ async def observe_relayed_messages(self):
         """
         Check the inbox for messages.
         """
-        messages = await self.api.messages_pop_all()
-        count = len(messages)
+        if self.address is None:
+            return
 
-        if count and not count & (count - 1):
-            logger.warning(
-                "Inbox might be full", {"count": count, **self.log_base_params}
+        for relayer, s in self.session_management.items():
```

Ruff (0.8.2)

Loop control variable `relayer` not used within loop body

Rename unused `relayer` to `_relayer`

(B007)